app/views/books/show.html.erb

```diff
@@ -1,6 +1,27 @@
 <h2><%= @book.title%> </h2>
-<p> Description: <%= @book.description %></p>
-<p> Year: <%= @book.year %></p>
-<p> Author: <%= link_to @book.author.first_name, @book.author %></p>
+<div>
+  <p> Description: <%= @book.description %></p>
```

Comment:
You should use `div` in every field instead of `<p>`. It will be easier to add the styles later.